001-update-metainfo.patch

```diff
@@ -0,0 +1,98 @@
+--- a/flatpak/tech.feliciano.pocket-casts.appdata.xml	2024-09-26 04:18:40.151643139 -0400
++++ b/flatpak/tech.feliciano.pocket-casts.appdata.xml	2024-09-26 04:19:23.269545738 -0400
+@@ -1,31 +1,80 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+-<component type="desktop">
++<component type="desktop-application">
+   <id>tech.feliciano.pocket-casts</id>
+-  <launchable type="desktop-id">tech.feliciano.pocket-casts.desktop</launchable>
++
+   <name>Pocket Casts</name>
++  <summary>Linux desktop player for Pocket Casts</summary>
+   <developer_name>Ricardo N Feliciano</developer_name>
+-  <summary>Online podcast player</summary>
++
++  <url type="homepage">https://github.com/FelicianoTech/pocket-casts-desktop-app</url>
++  <!-- Update the bugtracker link to upstream bugtracker when/if this Flatpak is verified -->
++  <url type="bugtracker">https://github.com/flathub/tech.feliciano.pocket-casts/issues</url>
++
+   <metadata_license>MIT</metadata_license>
+   <project_license>MIT</project_license>
+-  <url type="homepage">https://www.pocketcasts.com/</url>
++
++  <content_rating type="oars-1.1"/>
++
++  <recommends>
++    <control>pointing</control>
++    <control>keyboard</control>
++    <control>touch</control>
++  </recommends>
++
+   <description>
+-    <p>Pocket Casts is the world's most powerful podcast player. It's a great app to discover and listen to your favorite podcasts. Many blogs have deemed Pocket Casts the best podcatcher around.</p>
+-    <p>This is an Electron wrapper around the Pocket Casts web player. You need to have a pre-existing Pocket Casts account (it's cheap and worth the money) to use this app</p>
+-    <p>This wrapper is not verified by, affiliated with, or supported by Pocket Casts or Podcast Media LLC.</p>
++    <p>
++      Pocket Casts is the world's most powerful podcast player. It's a great app to discover and listen to your favorite podcasts. Many blogs have deemed Pocket Casts the best podcatcher around.
++      This is an Electron wrapper around the Pocket Casts web player. You need to have a pre-existing Pocket Casts account (it's cheap and worth the money) to use this app.
++      This wrapper is not verified by, affiliated with, or supported by Pocket Casts or Podcast Media LLC.
++    </p>
+   </description>
++
++  <categories>
++    <category>AudioVideo</category>
++    <category>Utility</category>
++  </categories>
++
++  <launchable type="desktop-id">tech.feliciano.pocket-casts.desktop</launchable>
++
+   <screenshots>
+     <screenshot type="default">
+-      <image type="source">https://raw.githubusercontent.com/felicianotech/pocket-casts-desktop-app/trunk/screenshot.png</image>
++      <caption>A screenshot of the main interface</caption>
++      <image>https://raw.githubusercontent.com/felicianotech/pocket-casts-desktop-app/v0.8.0/screenshot.png</image>
```

Comment:
This screenshot isn't suitable, please revert.

Those are the guidelines in case you're interested in updating the screenshot:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#screenshots